src/c360_client/dataset_cls/__init__.py

```diff
@@ -0,0 +1,233 @@
+import os
+import json
+import requests
+import getpass
+
+from c360_client.utils import get_boto_client
+
+
+class DatalakeClientDataset:
+    def __init__(
+        self, tenant=None, stage="prod", api_url=None, boto_endpoint_url=None, api_key=None
```

Comment:
Again, I'm not feeling so good about defaulting to `prod`

Reply:
see https://github.com/tra-sg/c360-python-client/pull/1#discussion_r549257480